File: TFCS-SPA/src/app/_models/surveyquestion.ts
import { SurveyOption } from './surveyoption';

export class SurveyQuestion {
    questionId: number;
    surveyId: number;
    questionName: string;
    shortQuestionName: string;
    questionOrder: number;
    questionType: string;
    surveyOptions: SurveyOption[];
    required: boolean;
}

File: TFCS.API/ModelsTest/SurveyQuestion.cs
using System;
using System.Collections.Generic;

namespace TFCS.API.ModelsTest
{
    public partial class SurveyQuestion
    {
        public SurveyQuestion()
        {
            SurveyOption = new HashSet<SurveyOption>();
        }

        public int QuestionId { get; set; }
        public int SurveyId { get; set; }
        public string QuestionName { get; set; }
        public string ShortQuestionName { get; set; }
        public int? QuestionOrder { get; set; }

        public virtual Survey Survey { get; set; }
        public virtual ICollection<SurveyOption> SurveyOption { get; set; }
    }
}
